docs/Debugging.md

```diff
@@ -8,24 +8,30 @@ next: testing
 ---
 
 ## Debugging React Native Apps
-To debug the javascript code of your react app do the following:
-
- 1. Run your application in the iOS simulator.
- 2. Press ```Command + D``` and a webpage should open up at [http://localhost:8081/debugger-ui](http://localhost:8081/debugger-ui). (Chrome only for now)
- 3. Enable [Pause On Caught Exceptions](http://stackoverflow.com/questions/2233339/javascript-is-there-a-way-to-get-chrome-to-break-on-all-errors/17324511#17324511) for a better debugging experience.
- 4. Press ```Command + Option + I``` to open the Chrome Developer tools, or open it via ```View``` -> ```Developer``` -> ```Developer Tools```.
- 5. You should now be able to debug as you normally would.
+To access the in-app developer menu, shake the iOS device or simulate a shake in the iOS Simulator by pressing `Control + Command + Z`.
 
 > Hint
 >
-> To debug on a real device: Open the file ```RCTWebSocketExecutor.m``` and change ```localhost``` to the IP address of your computer. Shake the device to open the development menu with the option to start debugging. 
+> To disable the in-app developer menu for production builds of your application, open the project in Xcode and navigate to `Product` → `Scheme` → `Edit Scheme...`(or press `Command + <`), select Run from the menu on the left, and change the Build Configuration to Release.
+
+### Reload
+Selecting Reload will reload the iOS application(this is the same as pressing `Command + R` in the iOS Simulator).
+
+### Chrome Developer Tools
```

Comment:
`⌘ + d` now opens the Dev Menu, and you have to select `Debug in Chrome` from there :smile: - can you update this?